push-images.sh

```diff
@@ -51,15 +54,28 @@ for v in "${versions[@]}"; do
     full_version=$(get_full_version $v)
     full_version_with_revision="${full_version}-${REVISION}"
 
-    # push relevant tags
-    echo "docker push \"${IMAGE_NAME}:${full_version}\""
-    docker push "${IMAGE_NAME}:${full_version}"
-    docker push "${IMAGE_NAME}:${full_version_with_revision}"
-    if [[ $full_version != *"-"* ]]; then
-      docker push "${IMAGE_NAME}:${v}"
-      if [ "${v}" == "${latest_version}" ]; then
-        docker push "${IMAGE_NAME}:latest"
-      fi
+    # build and push the multi-arch image
+    # unfortunately, when building a multi-arch image, we have to push it right after building it,
+    # rather than save locally and then push it. see https://github.com/docker/buildx/issues/166
+    if [ "${v}" == "${latest_version}" ]; then
```

Comment:
The changed if/else logic doesn't account for pre-release versions any more like  `1.0.0-rc1` where it will now also push the tag `1.0`. I don't think that we want that as it implies a production ready release in my opinion.